packages/ui/src/types/accountSettings/validator.ts

```diff
@@ -0,0 +1,17 @@
+export type InputEventType = 'blur' | 'change';
+
+export type ValidationMode = 'onBlur' | 'onChange' | 'onTouched';
+
+export interface ValidatorSpec {
```

Comment:
Nittish: Consider renaming `ValidatorOptions`

Reply:
Agreed, done in [7ff2acb](https://github.com/aws-amplify/amplify-ui/pull/2900/commits/7ff2acb46a95c1754c7e93bc9b7c98d3b527479f)